site/content/tutorial/08-motion/01-tweened/app-a/App.svelte

```diff
@@ -0,0 +1,30 @@
+<script>
+	import { writable } from 'svelte/store';
+
+	const progress = writable(0);
+</script>
+
+<style>
+	progress {
+		display: block;
+		width: 100%;
+	}
+</style>
+
+<progress value={$progress}></progress>
+
```

Comment:
I think I'm getting irrationally bugged by the lack of a '0%' button in this example. Do with this what you will.